plugin/trino-hudi/src/main/java/io/trino/plugin/hudi/HudiConfig.java

```diff
@@ -191,4 +194,17 @@ public HudiConfig setMaxOutstandingSplits(int maxOutstandingSplits)
         this.maxOutstandingSplits = maxOutstandingSplits;
         return this;
     }
+
+    @NotNull
+    public String getUserInputInstantTimeFormat()
+    {
+        return userInputInstantTimeFormat;
+    }
+
+    @Config("hudi.user-input-instant-time-format")
```

Comment:
I would rename to `hudi.commit-time-format` or something.

Reply:
removed